src/assets/articles/metaflow-resource-tracker.md

```
@@ -0,0 +1,303 @@
+
+The Metaflow card shows the resource usage of the step both at the process and
+system level (including CPU, memory, GPU, disk and traffic), plus the cloud
+resource recommendations based on the collected data enriched with Spare Cores
+data on cloud servers.
+
+In more detail, the card displays the following information (with example
+screenshots):
+
+- High-level hardware configuration of the server that executed the step.
+
+<img src="/assets/images/resource_tracker/resource-usage-server.webp" style="padding: 30px 0px 30px 30px;">
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)